osu.Game/Overlays/Comments/ReportCommentPopover.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Graphics.UserInterfaceV2;
+using osu.Game.Resources.Localisation.Web;
+using osuTK;
+
+namespace osu.Game.Overlays.Comments
+{
+    public class ReportCommentPopover : OsuPopover
+    {
+        private readonly DrawableComment comment;
+        private OsuEnumDropdown<CommentReportReason> reason = null!;
+        private OsuTextBox info = null!;
+        private ShakeContainer shaker = null!;
+
+        public ReportCommentPopover(DrawableComment comment)
+        {
+            this.comment = comment;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            Child = new FillFlowContainer
+            {
+                Direction = FillDirection.Vertical,
+                Width = 500,
+                AutoSizeAxes = Axes.Y,
+                Spacing = new Vector2(7),
+                Children = new Drawable[]
+                {
+                    new OsuSpriteText
+                    {
+                        Origin = Anchor.TopCentre,
+                        Anchor = Anchor.TopCentre,
+                        Text = UsersStrings.ReportTitle(comment.Comment.User?.Username ?? comment.Comment.LegacyName!),
+                        Font = OsuFont.Torus.With(size: 25),
+                        Margin = new MarginPadding { Bottom = 10 }
+                    },
+                    new OsuSpriteText
+                    {
+                        Origin = Anchor.TopCentre,
+                        Anchor = Anchor.TopCentre,
+                        Text = UsersStrings.ReportReason,
+                        Font = OsuFont.Torus.With(size: 20),
+                    },
+                    reason = new OsuEnumDropdown<CommentReportReason>
+                    {
+                        RelativeSizeAxes = Axes.X
+                    },
```

Comment:
The dropdown shouldn't autosize the popover when opening it, that results in bad UX:

https://user-images.githubusercontent.com/22781491/196821976-532ce76a-d5eb-4890-bcb8-bcce4cd8879f.mp4

Should work similar to multiplayer match settings.